arangod/ClusterEngine/ClusterTransactionState.cpp

```diff
@@ -171,8 +171,13 @@ Result ClusterTransactionState::abortTransaction(
   return {};
 }
 
-Result ClusterTransactionState::performIntermediateCommitIfRequired(
-    DataSourceId cid) {
+arangodb::Result ClusterTransactionState::triggerIntermediateCommit() {
+  TRI_ASSERT(false);  // should never be called!
+  return {};
```

Comment:
Should we just throw here (like we do in `performIntermediateCommitIfRequired`)?